netbox/extras/querysets.py

```diff
@@ -151,3 +152,14 @@ def _get_config_context_filters(self):
         )
 
         return base_query
+
+
+class ObjectChangeManager(Manager.from_queryset(RestrictedQuerySet)):
+
+    def get_queryset(self):
+        # Exclude any change records which refer to an instance of a model that's no longer installed. This
+        # can happen when a plugin is removed but its data remains in the database, for example.
+        app_labels = [
+            app.split('.')[-1] for app in settings.INSTALLED_APPS
+        ]
```

Comment:
```suggestion
        app_labels = [
            app.split('.')[0] for app in settings.INSTALLED_APPS
        ]
```

I think this is supposed to the the first element. I tested with netbox-dns, and this is what INSTALLED_APPS contains:

```
>>> settings.INSTALLED_APPS
['django.contrib.admin', 'django.contrib.auth', 'django.contrib.contenttypes', 'django.contrib.sessions', 'django.contrib.messages', 'django.contrib.staticfiles', 'django.contrib.humanize', 'corsheaders', 'debug_toolbar', 'graphiql_debug_toolbar', 'django_filters', 'django_tables2', 'django_prometheus', 'graphene_django', 'mptt', 'rest_framework', 'social_django', 'taggit', 'timezone_field', 'circuits', 'dcim', 'ipam', 'extras', 'tenancy', 'users', 'utilities', 'virtualization', 'wireless', 'django_rq', 'drf_yasg', 'netbox_dns.DNSConfig']
```

So the app_label for netbox_dns will be `netbox_dns` and not `DNSConfig`. I see that this is probably different for the django prefixed apps, where I'm guessing the app_label is actually the last part, but we do not generate changelogs for built-in django apps so not sure if it matters?

Reply:
Slightly more involved solution that also works, and is probably more correct.

```
# Add to imports
from django.apps import apps
from django.contrib.contenttypes.models import ContentType

app_labels = set(
    ContentType.objects.get_for_model(model).app_label for model in apps.get_models()
)
```

Tested working with netbox_dns. Not sure if there's any exotic plugins out that that could break though.

Reply:
Good call. Felt kinda dumb when I was writing it tbh. This is definitely cleaner.

Reply:
This has the added benefit of validating specific models, rather than just valid apps.

Reply:
Posting my slack message here as well for posterity:

> ContentType.objects.clear_cache() after app_labels = ... fixes tests for me